release-notes/swan-lake-beta4-release-note.md

```diff
@@ -298,6 +298,32 @@ float f13 = 0x1A.0;
 float f14 = 0x1A.0p4;
 ```
 
+- Disallowed intervening white spaces in the qualified identifier to avoid parsing an ambiguity between the ternary conditional expression and qualified identifier.
+  
+```ballerina
+import ballerina/io;
+
+public function main() {
+io:print("Ballerina"); // valid
+
+io : print("Ballerina"); // compilation error: intervening whitespaces are not allowed in qualified identifier
+}
+```
+
+With this, `x ? a : b:c` will be parsed now as `x ? a : (b:c)` since the colon with spaces is interpreted only as part of a conditional expression.
+
+- Reserve set of identifiers as ballerina keywords
+
+Identifiers `where`, `join`, `order`, `by`, `equals`, `ascending`, `descending`, `limit`, `outer`, and `select` are now considered as the ballerina reserved keywords. Further, the keyword `join` is allowed to be used in the method call expression as the method name.
+
+```ballerina
+// Previously allowed, now disallowed. As an alternative quoted identifier (`'limit`) can be used.
+int limit = 5;
+// Previously allowed, now disallowed. As an alternative quoted identifier (`string:'join()`) can be used.
+string b = string:join(" ", "hello", "world!");
+// Continue to allow `join` in method call expression
+myObject.join(obj); // allowed
+``` 
```

Comment:
Since we didn't specifically mention allowing these, I don't think we need to specify disallowing these? WDYT?

Reply:
We have mentioned that in Beta3 release note.

Reply:
Can you point to it please? Didn't we remove it as discussed in https://github.com/ballerina-platform/ballerina-dev-website/pull/2897#discussion_r725014020?

Reply:
Removed the section.